nzedb/Categorize.php

```diff
@@ -141,416 +142,416 @@ public function byGroup()
 		$group = $this->groupName();
 		if ($group === false) {
 			return false;
-		} else {
```

Comment:
Please revert all of this below, so it can be refactored properly to remove additional returns.